src/Admin/ValidationCounts.php

```diff
@@ -89,8 +89,9 @@ public function register() {
 	 * Enqueue admin assets.
 	 */
 	public function enqueue_scripts() {
-		/** This action is documented in includes/class-amp-theme-support.php */
```

Comment:
Please restore this comment.

Reply:
Upon checking it out, I don't think there is the documentation for `amp_register_polyfills` in `includes/class-amp-theme-support.php`. I think it should have been removed in some other changes.

Reply:
You're right. All instances of this comment should be replaced rather with a reference to `src/Admin/PairedBrowsing.php` which is where the phpdoc is now located.